src/app/app.component.spec.ts

```diff
@@ -1,18 +1,11 @@
 
 import { AppComponent } from './app.component';
+import { AppModule } from "./app.module";
 import { MockBuilder, MockRender } from 'ng-mocks';
-import { NavbarComponent } from './shared/navbar/navbar.component';
-import { FooterComponent } from './shared/footer/footer.component';
-import { RouterOutlet } from '@angular/router';
 
 describe('AppComponent', () => {
 
-  beforeEach(() => {
-    return MockBuilder(AppComponent)
-      .mock(NavbarComponent)
-      .mock(FooterComponent)
-      .mock(RouterOutlet)
-  })
+  beforeEach(() => MockBuilder(AppComponent, AppModule))
```

Comment:
the simplest way is to pass the module where the component has been declared.

Reply:
yep, got it, i guess I had done that once somewhere in the app, was mixing and matching to show all variations!